Automattic/woocommerce-payments
# Fix details not displayed in deposits overview

Fixes #962 

#### Changes proposed in this Pull Request

Updated the `SummaryNumber` in deposits overview to use `children` prop instead of `prevLabel` to show additional details. Also added a new css class to keep using the old font color.

I opted to use the children prop inside the spread instead of writing in the markup to avoid making another `if`.

This works using the new children prop in `SummaryNumber` added in https://github.com/woocommerce/woocommerce-admin/pull/6748. Is now merged and released in `@woocommerce/components@6.1.0` but unreleased in `@woocommerce/admin-library`.

It will only work with new releases of WCAdmin, in older versions the details will remain hidden like now.

![image](https://user-images.githubusercontent.com/7670276/115213825-31a78200-a102-11eb-8c14-7db9d497c517.png)
_Screenshot working_

#### Tests and old version support
After an internal talk (p1619421231464600-slack-C01B8KNUYSW), we agreed to keep using an older version of `@woocommerce/components` to keep testing compatibility and have simple cases like this uncovered. Instead of bumping up the dependencies versions and maybe miss some incompatibilities and cause errors to users with the older supported versions. And in this case, also we decide to not implement an alternative solution for older versions on WCAdmin, because is something not critical, we can just show it to newer versions.

#### Testing instructions

* Install and activate WCAdmin from main branch.
* Navigate to **Payments > Deposits**.
* Show should see overview details like the screenshot above.

-------------------
